app/forms/PropertyDetailsForms.scala

```diff
@@ -51,41 +56,45 @@ object PropertyDetailsForms {
     """^(?!\.)("([^"\r\\]|\\["\r\\])*"|([-a-zA-Z0-9!#$%&'*+/=?^_`{|}~]|(?<!\.)\.)*)
       |(?<!\.)@[a-zA-Z0-9][\w\.-]*[a-zA-Z0-9]\.[a-zA-Z][a-zA-Z\.]*[a-zA-Z]$""".r
   val supportingInfoRegex = """^[A-Za-z0-9\s\.\,\']*$"""
-  val titleRegex = """^[A-Za-z0-9\s\.\,\']*$"""
+  val titleRegex = "^[A-Za-z0-9 ,.&']{1,40}$"
```

Comment:
Please follow best practices to name constants. Would it be worth moving these to constant file?

Reply:
Done the code changes